pages/index.js

```diff
@@ -113,8 +117,17 @@ export default function HomePage() {
     await mutate();
   }
 
+  if (status !== "authenticated") {
+    return (
+      <AuthBar>
+        <h2>Please log in to access this page</h2>
+        <AuthButtons />
+      </AuthBar>
+    );
+  }
```

Comment:
Since you have already protected the whole app (in pages/_app.js) with the `<Auth>` component you don't need to check it here again. Without authentication you will never reach this page.